pkg/api/authz/authz.go

```diff
@@ -119,15 +120,21 @@ type Authorizer struct {
 	apiResources *pathMatcher
 	uiResources  *pathMatcher
 	acrHelper    *accesscontrolrule.Helper
+	Bootstrapper interface {
+		BootstrapEnabled(ctx context.Context) (bool, error)
+	}
```

Comment:
This field doesn't need to be exported.

```suggestion
	bootstrapper interface {
		BootstrapEnabled(ctx context.Context) (bool, error)
	}
```